mackup/applications/datagrip.cfg

```diff
@@ -0,0 +1,12 @@
+[application]
+name = DataGrip
+
+[configuration_files]
+Library/Application Support/DataGrip2017.3
+Library/Preferences/DataGrip2017.3
+Library/Application Support/DataGrip2018.1
+Library/Preferences/DataGrip2018.1
+Library/Application Support/DataGrip2018.2
+Library/Preferences/DataGrip2018.2
+Library/Application Support/DataGrip2018.3
+Library/Preferences/DataGrip2018.3
```

Comment:
Can you finish the file with a newline char?

Reply:
Done